### DISCUSSION TITLE
Enable/Disable Voicemail to Email Notification

### DISCUSSION THREAD

**Discussion by vaijanaths007** [kctrey/wxcadm#5] [Ideas]
Hi Trey,

I have added below code to person.py to my local copy. Maybe you can consider adding it to your repository:

`
[vm_notification.txt](https://github.com/kctrey/wxcadm/files/10739065/vm_notification.txt)
`

**Comment by kctrey** [comment]
There was a reason I originally left that out, because they really serve the same purpose for most people, and I felt like the two options were confusing. That said, I do see value in it when you want to not have the VM content delivered externally. I will add this to the [dev] branch now and it will release with 3.2.0

**Reply by vaijanaths007** [reply]
Thanks Trey. Really appreciate it.

**Comment by kctrey** [comment]
@vaijanaths007 I released v3.2.0 this morning with your methods included.
